setup.py

```diff
@@ -47,8 +47,8 @@ def run(self):
       license='MIT',
       packages=find_packages(),
       package_data={'gymfc': [
-          "envs/assets/gazebo/models/quadcopter_attitude_control/*.config",
-          "envs/assets/gazebo/models/quadcopter_attitude_control/*.sdf",
+          "envs/assets/gazebo/models/attitude_control_training_rig/*.config",
```

Comment:
Nice catch, does this prevent the correct files being installed when *not* installed in development mode?

Reply:
mhh, not sure I understand. I think this did not work before, as the files are located in attitude_control_training_rig not in quadcopter_attitude_control

Reply:
Correct this is a bug, reason I think it went unnoticed is because if you install via pip in development mode (i.e., pip3 install -e .) this isn't needed.